app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsAlertDialog.kt

```diff
@@ -0,0 +1,71 @@
+package com.github.se.cyrcle.ui.parkingDetails
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.rememberAsyncImagePainter
+import com.github.se.cyrcle.R
+import com.github.se.cyrcle.ui.theme.atoms.Text
+
+@Composable
+fun ParkingDetailsAlertDialog(
+    onDismiss: () -> Unit,
+    onAccept: () -> Unit,
+    newParkingImageLocalPath: String,
+) {
+  AlertDialog(
+      modifier = Modifier.testTag("parkingDetailsAlertDialog"),
+      onDismissRequest = { onDismiss() },
+      title = { Text(stringResource(R.string.card_screen_confirm_upload)) },
+      text = {
+        Column(
```

Comment:
I don't think you need to wrap the image in a column. You can just add a container color parameter to the Alert Dialog to use surface bright